README.md

```diff
@@ -2654,7 +2691,7 @@ Other Style Guides
     ```
 
   <a name="whitespace--max-len"></a><a name="18.12"></a>
-  - [19.12](#whitespace--max-len) Avoid having lines of code that are longer than 100 characters (including whitespace). Note: per [above](#strings--line-length), long strings are exempt from this rule, and should not be broken up. eslint: [`max-len`](https://eslint.org/docs/rules/max-len.html)
+  - [19.12](#whitespace--max-len) Avoid having lines of code that are longer than ~~100~~ 120 characters (including whitespace). Note: per [above](#strings--line-length), long strings are exempt from this rule, and should not be broken up. eslint: [`max-len`](https://eslint.org/docs/rules/max-len.html)
```

Comment:
How about just having 120? We dont need to convey a message "U know at some point it was 100, but now its 120"

Reply:
I agree with having it set to maximum.

Reply:
@abhinavmsra Okay, I'm fine with anything :)
@vtno `~~` means strikethrough in markdown.